docs/design/autoware-interfaces/ad-api/list/api/planning/docs/factors-lane-change.drawio.svg

```diff
@@ -0,0 +1,176 @@
+<svg host="65bd71144e" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="804px" height="323px" viewBox="-0.5 -0.5 804 323" content="&lt;mxfile&gt;&lt;diagram id=&quot;OOft0IhOQVC92sn3leyt&quot; name=&quot;Page-1&quot;&gt;7Zpbb5swFIB/TV4rwJDLY5qmF2lbo6XaHic3uMGag5FxLt2vnwmGgE0SSkyTVqVShQ/m2D7fOfaxSQeMFps7BqPgO/UR6TiWv+mAm47jDHo98T8RvKYC0B2kgjnDfiqyd4Ip/oek0JLSJfZRXKrIKSUcR2XhjIYhmvGSDDJG1+VqL5SUW43gHGmC6QwSXfob+zyQUjsbRvLgHuF5IJvuO3LAC5hVliOJA+jTdUEExh0wYpTy9G6xGSGS2C6zS/re7Z6neccYCnmdFxw3fWMFyVIOTnaMv2ajRaE/TIwmSiENhfA64AsiSra4jTmjf3MrACF5wYSMKKFs+zZAnu1aVl6z8MTaXuKJaEAy9rb1IOOFsj4kOUrklyjJAd4hukCcvYoK6x0bT9o7KFDJZAwRyPGqzBZKF5nn6vIWJhSLnjiW9GYvYymd2elZZRUxXbIZkm8VWRxTZCuKhGHmiGuKxE1h2DvRFvUe7MapXzolV7Gt1RBS/5gic5DMh+YHgwS6hiBpisxB8ppA8mEcIF9i2kvM+XjEnL6psFIVmSPWPU5MrMpRcrvYzJP85SrG8zC+4gyGcUQZFwaj4dUMsj+gzC9KuofYeCVoxVKmLIf5oldeDqVjrBDjWOQZ3+AzIhMa46Ql8fiZck4XhQpDIrokHnAaCSmUpRlKmk8aJTi6lx3Y60KJLrQ56EQbZZLL6GQrVcHJbKvCy1xrv0OVEB7g1X9ThM0IjGM8K3Mpxdulh1Q+WWXGVo3YOJ9QFRmcBT2dibDmVBb1tWkvhHRwUqsM1bSfUtavCas2hbp+6L1h4hBvir2IMuZ9M8ELDbN81+7LcqHey/aqSqBvreTPTIj3y77i6gHuthTfXq8V3+lX+M7gbL6jT2LDyeTn43B0//DjThs/o8vQzyesdYA5mkZwO7S1WJBqmqP5/F5B33Zawt+rmM4P4q+c7IXwFpNyHlWIlMH2qu07A913umfznR74EAbKdr1nCK6BHlyjp4dfw6dLiC3XPV9sZZnE1+mOuWys6emOpqi9052u/YX91H2tkg8BryF2VZEzaA+784X9ROxd9eCoKXZVUZvYwZuwV26QL52LFo5OQy4954gig1xOPb+9dCjGDiqO0jUIpcYm/mNDsQ1BOUrXIJTeiVBK53u7nc7Bo5EL4XX0kK7pitNmEF30hqt7/CirvcMMV3fcwl6dMh7QOQ0hGe+kxrbuO0MfdPziRj7bzFzISYe+XZ0+PU4m45uzO5W6i8896F28Sv/IONQsIgbCq9J55QtTMaOXIu3rkfqRaYF9f+upVXau6XX1TQ3U1NnTTV1laWDC0np6cP15Le2pn/Xe09L6mj/6vJZ2lWQKgHe0tP7tQZ9PP42lu+oGrz1Li+Luh3ppGrX7tSMY/wc=&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
There are two boxes of `ACTIVATING`. Is that intended? I think the right one is `ACTIVATED`.
![image](https://user-images.githubusercontent.com/31987104/179395801-306a7d20-77c6-41c8-922d-026ff95f2365.png)

Reply:
Fixed https://github.com/autowarefoundation/autoware-documentation/pull/154/commits/86a0a5cf78c465a17df2f189c423e3a7699ba56a.